pyproject.toml

```diff
@@ -1,3 +1,7 @@
+[project]
+name = 'pyjnius'
+requires-python = ">= 3.7"
```

Comment:
Pythonista 3.9 will end-of-life this week.  Perhaps this should be 3.10 so users are not building solutions on a slower and less secure base.